index.html

```diff
@@ -46,6 +46,204 @@ <h1>I'm Naan.
             </div>
         </headline>
         <!-- END OF HEADLINE SECTION-->
+
+        <!--Works Section-->
+        <!--Work 1-->
+        <div class="primary_text">
```

Comment:
I have observed that your portfolio's work section has a background `#fff`. OK let me explain it again:
The `background-color` of the `<body>` is `#F7F7F9`. This is the same background that is behind the work section's div and work-section does not have any of its own background. 
Since you have work-section background and work cards background as white: Let's compare both the Figma and Your design

| Your Design   | FIGMA Design  |
| ------------- | ------------- |
| ![image](https://user-images.githubusercontent.com/61361037/219896408-e6dd13bb-5fe9-42d4-b474-19da2742ed34.png)| ![image](https://user-images.githubusercontent.com/61361037/219896346-4d837284-f433-424a-8c1d-1997222d802a.png)|

Reply:
@RaoAkif Thank you for the thorough review. how should I solve this specific issue?